### PR TITLE
bugfix/#5276/[Green city. Events] The buttons 'Cancel'/'Preview'/ 'Publish' are not aligned according to mock up on the page 'Create event' 

### PR DESCRIPTION
**Pull request by AlexW8719**

### CODE REVIEW

**Comment by AlexW8719** [comment]
made some change in scss (create-edit-events.component)

File: src/app/main/component/events/components/create-edit-events/create-edit-events.component.scss [hunk 1]
```diff
@@ -140,6 +140,8 @@
   padding: 40px 0 60px;
   display: flex;
   justify-content: flex-end;
+  align-items: start;
+  max-width: 1640.4px;
```
Comment (atsa21) [inline src/app/main/component/events/components/create-edit-events/create-edit-events.component.scss]: 1640.4px don't match principle of Pixel Perfect Design, it is better to write value without decimal values

Comment (AlexW8719) [reply src/app/main/component/events/components/create-edit-events/create-edit-events.component.scss]: Ok, I'll change to 1640

File: src/app/main/component/events/components/create-edit-events/create-edit-events.component.scss [hunk 2]
```diff
@@ -140,6 +140,8 @@
   padding: 40px 0 60px;
   display: flex;
   justify-content: flex-end;
+  align-items: start;
```
Comment (atsa21) [inline src/app/main/component/events/components/create-edit-events/create-edit-events.component.scss]: With 'start' console display warning. Need to use flex-start
image

**Review by atsa21 — APPROVED** [review]

**Review by KKatiaa — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Review by SokhaTaras — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Review by marianaharnyk — APPROVED** [review]

**Pull request merged**